civictechprojects/models.py

```diff
@@ -48,6 +48,7 @@ class Project(models.Model):
     project_name = models.CharField(max_length=200)
     project_url = models.CharField(max_length=2083, blank=True)
     project_links = models.CharField(max_length=5000, blank=True)
+    project_date_modified = models.DateTimeField(auto_now=True)
```

Comment:
What value will project_date_modified be in existing projects that haven't been updated?

Reply:
I believe that when a model is first created, it will be treated as having been "updated", since the "auto_now" attribute specifies that the date field will be set when the model is saved.  Is that correct?